ui/segment.reel/segment.css

```diff
@@ -0,0 +1,12 @@
+.montage-Segment {
+    /* to vertical align text labels */
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+
+    -webkit-box-align: center;
+    -webkit-align-items: center;
+    -ms-flex-align: center;
+    align-items: center;
+}
```

Comment:
Fair enough... kind of. I'm not thrilled about a tool deciding which browsers our code supports, especially when someone reading the code isn't going to know what tool generated it, what our settings were for this tool, and generally why it generated the values it generated. Can we configure autoprefixer to just generate code for the browsers we support, like a human would have done? What happens if we're committed to supporting browsers that autoprefixer decides to drop because of dwindling market share?

In any case, it may be worth putting a comment in this .css files, even if it's just one at the top explaining that autoprefixer was used to generate the browser variant prefixes. Regardless of what tool is used, we humans are responsible for making sure the code we commit isn't generating too many WTF's (like my WTF is -webkit-box doing here).